src/redux/reducers/index.ts

```diff
@@ -52,11 +51,55 @@ const trapVisitPostPersistConfig = {
   version: 1,
   storage: AsyncStorage,
 }
+// SHOULD POSSIBLY USE SECURE STORE INSTEAD OF ASYNC STORAGE
+//import SecureStore from 'expo-secure-store'
+
+export interface Options {
+  replaceCharacter?: string
+  replacer: (key: string, replaceCharacter: string) => string
+  authenticationPrompt?: string
+  keychainAccessible?: any
+  keychainService?: string
+  requireAuthentication?: boolean
+}
+
+// function createSecureStorage(options = {} as Options) {
+//   const replaceCharacter = options.replaceCharacter || '_'
+//   const replacer = options.replacer || defaultReplacer
+
+//   console.log('SecureStore', SecureStore)
+
+//   return {
+//     getItem: (key: string) => SecureStore.getItemAsync(key, options),
+//     setItem: (key: string, value: any) =>
+//       SecureStore.setItemAsync(key, value, options),
+//     removeItem: (key: string) => SecureStore.deleteItemAsync(key, options),
+//   }
+//   return {
+//     getItem: (key: string) =>
+//       SecureStore.getItemAsync(replacer(key, replaceCharacter)),
+//     setItem: (key: string, value: any) =>
+//       SecureStore.setItemAsync(replacer(key, replaceCharacter), value),
+//     removeItem: (key: string) =>
+//       SecureStore.deleteItemAsync(replacer(key, replaceCharacter)),
+//   }
+// }
+// function defaultReplacer(key: string, replaceCharacter: string) {
+//   return key.replace(/[^a-z0-9.\-_]/gi, replaceCharacter)
+// }
+
+// const userCredentialsPersistConfig = {
+//   key: 'userCredentialsPersistConfig',
+//   version: 1,
+//   storage: createSecureStorage(),
+// }
+
```

Comment:
Did not want to delete this as I did not write it, but omitting this code block does not affect sign in flow